it/server/src/test/java/com/walmartlabs/concord/it/server/ITConstants.java

```diff
@@ -9,9 +9,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
```

Comment:
A bit noisy, could you please remove those unnecessary changes?

Reply:
yes removed :)